### ISSUE TITLE
support for the bzip option

### ISSUE THREAD

**Issue by SirParadox** [oguna/Blender-XFileImporter#5]
TL;DR Support the bzip (MSZip) format of the .x files

I was pleased to find this plugin, got my correct blender version installed, this configured, and shizam it worked on my game's ancient .x files.  My game I inherited almost 15 years ago.  As far as I know all my .x files were created in 3dsmax (2005) along with my bmp and dds files. 

Moving on I went to view a few other .x files and ran into a wall.  Some of the games late development .x files were bzip encoded.  Apparently thats not bzip like we all know bzip, but 'MSZip Compressed Binary File'.

I read this on https://paulbourke.net/dataformats/directx/

I've looked at the header in the file and I dont see a spot where the typical  PK starts and the zip starts... so I can't just hexedit my file, remove the xof..... bits, unzip, restore xof bits with bin and move on.  

1. Any ideas how to extract the Zip data to a binary/text .x file?
2. Any chance you might natively support reading this format?
3.
